sns/parity_math.c

```diff
@@ -32,7 +32,18 @@
 #define M0_TRACE_SUBSYSTEM M0_TRACE_SUBSYS_SNS
 #include "lib/trace.h"
 
-#define ir_invalid_col_t UINT8_MAX
+#define ir_invalid_col_t   UINT8_MAX
```

Comment:
This is in the existing code, but we may also to fix it:
* If possible, use `enum` instead of `define` to define a constant value.
* if "define" is used, the name of this MACRO should be CAPITALIZED.

Reply:
Done